tests/inputs/models/inj_beam1/macros/general.macros.madx

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
shouldn't this be somewhere in the repo already?

Reply:
It is in `omc3.model.madx_macros`, should be taken from there.